baselines/lamb/jax/submission.py

```diff
@@ -0,0 +1,245 @@
+"""Submission file for a LAMB optimizer with warmup+cosine LR in Jax."""
+
+import functools
+
+# isort: off
+# We have to turn off isort here to resolve a conflict between isort and yapf.
+from typing import (Any, Callable, Dict, Iterator, List, Optional, Tuple, Union)
+# isort: on
+
+from flax import jax_utils
+import jax
+from jax import lax
+import jax.numpy as jnp
+import optax
+
+from algorithmic_efficiency import spec
+
+_GRAD_CLIP_EPS = 1e-6
+
+
+# Forked from
+# https://github.com/deepmind/optax/blob/master/optax/_src/alias.py.
```

Comment:
besides the default eps of 1e-8 vs optax's 1e-6, was there a reason for forking instead of calling `optax.lamb` directly below? if not, can we just call `optax.lamb(..., eps=1e-8)` ?